# IMPORTANT: after changing this file, you have to build the workspace again. otherwise the params are not updated.
graphviz_visualizer_node: # same node name specified in launch file
  ros__parameters:
    # IMPORTANT: the index in the array indicates the node id
    node_names: [
      'Node 1',
      'Node 2',
      'Node 3',
      'Node 4',
      'Node 5',
      'Node 6',
      'Node 7',
      'Node 8',
      'Node 9',
      'Node 10',
      'Node 11',
    ]